### PR TITLE
Brought releases/v1 in sync with main

### PR DESCRIPTION
**Pull request by t-dedah**
Synced releases/v1 with main as part of this WI https://github.com/github/azure-integration/issues/342

### CODE REVIEW

**Comment by bishal-pdMSFT** [comment]
@t-dedah please remove Kanika from codeowners file

**Comment by bishal-pdMSFT** [comment]
> @t-dedah please remove Kanika from codeowners file

Ah you are already doing that

File: .github/workflows/add-labels.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: add-labels
```
Comment (bishal-pdMSFT) [inline .github/workflows/add-labels.yml:1]: Is this for adding to ACE's health dashboard?

File: .github/workflows/add-labels.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+on: 
+  workflow_dispatch:
```
Comment (bishal-pdMSFT) [inline .github/workflows/add-labels.yml:3]: should this be a cron schedule?

File: .github/workflows/add-labels.yml (hunk 1 above)
Comment (t-dedah) [reply .github/workflows/add-labels.yml:1]: Yes Bishal

File: .github/workflows/add-labels.yml (hunk 2 above)
Comment (t-dedah) [reply .github/workflows/add-labels.yml:3]: No it's a one-time thing, this is just to create new labels with particular colours.

Comment (bishal-pdMSFT) [reply .github/workflows/add-labels.yml:3]: Oh! I thought it will automatically add a label as well :D

**Review by bishal-pdMSFT — APPROVED** [review]

**Pull request merged**